README.md

```diff
@@ -1,175 +1,404 @@
-# postgres-mcp: PostgreSQL Tuning and Analysis Tool
+# Postgres Pro | MCP Server
 
-postgres-mcp is a Model Context Protocol (MCP) server that helps you analyze, optimize, and monitor your PostgreSQL databases. It allows AI assistants like Claude to analyze query performance, recommend indexes, and perform health checks on your database.
+[ [Quick Start](#quick-start) | [Intro Blog Post](https://www.crystaldba.ai/blog) | [Discord Server](https://discord.gg/4BEHC7ZM) ]
+
+## Overview
+
+*Postgres Pro* is a Model Context Protocol (MCP) server that helps you query, analyze, optimize, and monitor your PostgreSQL databases.
+It allows AI assistants and agents such as Claude or Cursor to analyze query performance, recommend indexes, and perform health checks on your database.
+
+Postgres Pro provides support along the full software development lifecycle—from code generation through testing, deployment, and production maintenance and tuning.
+
+*DEMO VIDEO PLACEHOLDER*
+
+## Contents
+
+- [Features](#features)
+- [Quick Start](#quick-start)
+- [Usage Examples](#usage-examples)
+- [MCP Server API](#mcp-server-api)
+- [Related Projects](#related-projects)
+- [Frequently Asked Questions](#frequently-asked-questions)
+- [Technical Notes](#technical-notes)
+- [Postgres Pro Development](#postgres-pro-development)
 
 ## Features
 
-postgres-mcp provides several powerful tools for database optimization:
+Postgres Pro offers a set of tools to help you query, analyze, and optimize your Postgres database.
+It provides:
+
+- **Schema Information**.
+  Help your AI Agent generate SQL reliably and successfully with detailed schema information of your database objects—including tables, views, sequences, stored procedures, triggers.
+
+- **Protected SQL Execution**.
+  Work fast or safe, as needed:
+  - *Unrestricted Mode:* Provide full read/write access in development environments. Let your AI agent modify data, change the schema, drop tables, whatever you need it to do.
+  - *Restricted Mode:* Limit access in production environments with checks to ensure read-only operations and limits on resource consumption. These restrictions may become more elaborate in the future.
+
+- **Index Tuning**.
+  Ensure your SQL queries run efficiently and return quickly.
+  Find tuning targets, validate AI-generated suggestions, or generate candidates using classical index optimization algorithms.
+  Simulate how Postgres will after adding indexes using the explain plans together with [hypothetical indexes](https://hypopg.readthedocs.io/).
+
+- **Database Health**.
+  Check buffer cache hit rates, identify unused/duplicate indexes, monitor vacuum health, and more.
```

Comment:
Indexs first, then buffer cache hits later -- lots of people won't know what that implies.